recorder/src/test/groovy/com/focusit/recorder/specs/InternalRecorderBuilderSpec.groovy

```diff
@@ -0,0 +1,18 @@
+package com.focusit.recorder.specs
+
+import com.focusit.jsflight.recorder.internalevent.InternalEventRecorderBuilder
+import spock.lang.Specification
+
+class InternalRecorderBuilderSpec extends Specification {
+    def "Several builders return same recorder for same file"() {
```

Comment:
I don't think it is right case.

As for me, building another recorder for the very same path should lead to an IllegalArgumentException. It is wrong behaviour.